readme.MD

```diff
@@ -1,5 +1,5 @@
 # OBS-VirtualCam
-obs-virutalcam is a plugin for obs-studio , transforming the output video to a virtual directshow device.
+OBS-Virtualcam is a plugin for obs-studio , transforming the output video to a virtual directshow device.
```

Comment:
i'd suggest using `OBS-VirtualCam` here

Reply:
Changed according to your suggestion, I think you're right 👍